docs/source/advanced/validator.rst

```diff
@@ -0,0 +1,61 @@
+Validate Command
+-----------------
+
+New in version 1.6.7
```

Comment:
Coming in version 1.6.7? Haha! However, we should figure out how we're going to promise things like this that aren't in a release yet and may not make the release. Part of why I considered just putting it in as an 'Advanced Topic'. Let's discuss this detail more and come up with a plan.

Reply:
That's fine with me, I was largely just copying other files at this point.  I can remove this line for now until we decide when this will become part of a release